packages/react/src/transactions/adapters/responders/MirrorResponder.ts

```diff
@@ -0,0 +1,58 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client';
+import {
+  PublicationByTxHashDocument,
+  PublicationByTxHashQuery,
+  PublicationByTxHashQueryVariables,
+} from '@lens-protocol/api-bindings';
+import { CreateMirrorRequest } from '@lens-protocol/domain/use-cases/publications';
+import {
+  BroadcastedTransactionData,
+  ITransactionResponder,
+  TransactionData,
+} from '@lens-protocol/domain/use-cases/transactions';
+
+import { PublicationCacheManager } from '../PublicationCacheManager';
+
+export class MirrorResponder implements ITransactionResponder<CreateMirrorRequest> {
```

Comment:
nit: `CreateMirrorResponder` maybe... not strong about it. It's also clear as it is.

Reply:
make sense, fixed